src/main/java/ru/sbtqa/tag/pagefactory/Page.java

```diff
@@ -825,7 +825,7 @@ public String getElementTitle(WebElement element) {
      * find corresponding element or element type is set incorrectly
      */
     public WebElement getElementByTitle(String title) throws PageException {
-        if (!usedBlock.isEnabled()) {
+        if (usedBlock != null) {
```

Comment:
You're breaking down the current logic while fixing NPE. 
Change condition as following:
```
 if (usedBlock != null && !usedBlock.isEnabled()) {
```